lib/helpers/backend/paginate.ts

```diff
@@ -0,0 +1,5 @@
+export function paginateResults(array: any[], length: number, page: number) {
```

Comment:
If you do `paginateResults<T>(array: T[], length: number, page: number)` instead, it keeps the return type of the function correct without requiring any change from the caller

Reply:
With how it is currently, `paginateResults` always returns `any[]`, regardless of what is passed in

Reply:
Good tip thank you